docker/images/node/Dockerfile

```diff
@@ -20,5 +20,8 @@ COPY gulpfile.babel.js gulpfile.babel.js
 
 # we use `--no-optional` because some optional npm dependencies fail on install
 RUN npm install -g gulp-cli
+# `npm install gulp-imagemin` don't install it's dependencies sometimes.
+# Took decision from SO: https://goo.gl/3uXZxw
+RUN npm install imagemin-jpegtran imagemin-svgo imagemin-gifsicle imagemin-optipng --save
```

Comment:
Let's move these dependencies to the `package.json`

Reply:
@artemiy312 , i don't want to create mess in `package.json`.
Now it's the good place to look on dependencies and it's versions
But the huge group of `imagemin-*` says nothing useful and takes 50% of `package.json` file.
Nothing useful, because it should be installed recursively from `gulp-imagemin`.

So, it's good to put it to one-lined npm install with it's own clear comment